server/src/main/java/org/elasticsearch/common/util/concurrent/EsThreadPoolExecutor.java

```diff
@@ -121,6 +121,15 @@ public Stream<Runnable> getTasks() {
         return this.getQueue().stream().map(this::unwrap);
     }
 
+    /**
+     * Basic implementation only returns instantaneous utilisation
+     *
+     * @return The percentage of the maximum threads that are in use right now
+     */
+    public double getUtilisation() {
+        return (double) getActiveCount() / getMaximumPoolSize();
+    }
```

Comment:
Not sure if it's worthwhile having this, or just omitting this metric for non-timed pools altogether. Could be misleading.

Reply:
Yeah, I'd prefer to omit it too, too different.